docs/index.rst

```diff
@@ -32,6 +32,7 @@ Understand messages, hold conversations, and connect to messaging channels and A
    user-guide/validate-files
    user-guide/configuring-http-api
    user-guide/how-to-deploy
+   user-guide/setting-up-ci-cd
```

Comment:
How about putting this right after testing? I imagine you might want to setup ci/cd before even deploying